translations/de.json

```diff
@@ -0,0 +1,12 @@
+{
+  "language": {
+    "de": "Deutsch"
+  },
+  "panel": {
+    "states": "Zustände",
+    "map": "Karte",
+    "logbook": "Logbuch",
+    "history": "Vergangenheit",
```

Comment:
That looks like machine translated.

Reply:
Thanks for the very subjective feedback. However, they are not machine translated. And if you look at #477 , you will see all the same translations, except for one.

And the reason that that one word is not matching up, is because the word in #477 is not a proper translation of the source word. And that doesn't mean it is necessarily incorrect for the desired outcome, but that is a design/product decision that needs to be made.

Either you go for proper source words, that will result in (objectively) proper/correct translations in other languages. OR you choose to go the route where you accept (subjective) interpretations of the source word as translations.

I commented similar feedback in #483 . I'm fine either way, but if you guys are closing issues/commits based on 1 individuals subjective (and incorrect) feedback, without specific argumentation or design/product decisions or guidelines, I'm not going to invest any more time contributing, and will just update my own correct translations in my local fork.